ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/MSVS_2019/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+################################
+# Visual Studio Community 2019 #
+################################
+
+- name: Test if VS 2019 is installed
+  win_stat:
+    path: 'C:\Program Files (x86)\Microsoft Visual Studio\2019'
+  register: vs2019_installed
+  tags: MSVS_2019
+
+- name: Download Visual Studio Community 2019
+  win_get_url:
+    url: 'https://aka.ms/vs/16/release/vs_community.exe'
+    dest: 'C:\TEMP\vs_community.exe'
```

Comment:
Could this be changed to `C:\temp\vs_community_2019.exe` ? Otherwise It'll overwrite the MSVS_2017 `vs_community.exe`.

Reply:
Updated.